.travis.yml

```diff
@@ -1,7 +1,10 @@
+dist: xenial
```

Comment:
[ask] @dargueta Is it necessary to specify `dist` field with `xenial` ?

Reply:
To get Python 3.7, yes.

Reply:
Sure.